### PR TITLE
Alert and button

### PR DESCRIPTION
**Pull request by kotalwar-krishna**
**Alert component**
-I created Alert with text, link and icon
-alerts realeted CSS you can find in Alert.css
**Button**
-Button realeted CSS you can find in Button.css

Please review my code and comment below.
**Note:-** Please do not consider colors in your review i will change when i will refactor all the Component library and for reusable color i will use Variable.

Screenshots:-
Alert
button_icon float
button_o l
button_s

### CODE REVIEW

**Review by Raghav888 — APPROVED** [review]
Approved

File: components/button.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
+
+.btn {
+    margin: 1rem;
+    height: fit-content;
+    width: fit-content;
+    border-radius: 2em;
```
Comment (ozairkhan2810) [inline components/button.css]: maintain consistency in units

Comment (kotalwar-krishna) [reply components/button.css]: ok i wii do

**Review by ozairkhan2810 — APPROVED** [review]
code looks good,

File: components/button.css (hunk 1 above)
Comment (kotalwar-krishna) [reply components/button.css]: some typo was there

Comment (kotalwar-krishna) [reply components/button.css]: i done that but not commited i will commit

**Pull request merged**